content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
```

Comment:
Did https://github.com/istio/istio/pull/14883 make it into 1.2.0 or 1.2.1?  If the latter:

- Eliminate 503 errors caused by Envoy not being able to talk to the SDS Node Agent after a restart ([Issue 14853](https://github.com/istio/istio/issues/14853)).

Reply:
#14883 made it before the empty commit (so 1.2.0)